metrics/wait.io.table/table.go

```diff
@@ -0,0 +1,236 @@
+// Copyright 2022 Block, Inc.
+
+package waitiotable
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/cashapp/blip"
+)
+
+const (
+	DOMAIN = "wait.io.table"
+
+	OPT_EXCLUDE        = "exclude"
+	OPT_INCLUDE        = "include"
+	OPT_TRUNCATE_TABLE = "truncate-table"
+	OPT_ALL            = "all"
+
+	TRUNCATE_QUERY = "TRUNCATE TABLE performance_schema.table_io_waits_summary_by_table"
+)
+
+var (
+	metric_names = []string{
```

Comment:
`CamelCase` is the norm in Go and Blip. We can also give this and `metic_map` below more intention-revealing names, maybe... `columnNames` (the list) and `columnExists` (the map since that's what it's used for: to see if it exists).